masschallenge/django-accelerator
# [AC-9102] Create PartnerJudgingInstructions model

## [AC-9102](https://masschallenge.atlassian.net/browse/AC-9102)

**Changes introduced**
- Added PartnerJudgingInstructions model

**How to test**
- Check that def of done is satisfied [AC-9102](https://masschallenge.atlassian.net/browse/AC-9102)
